pkg/apis/eventing/logkey/constants.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logkey
+
+const (
+	kNative = "knative.dev/"
```

Comment:
the compiler says "this is not a constant" for `const ClusterProvisioner = v1alpha1.Kind("ClusterProvisioner").String()`

Reply:
Pseudocode example :) I'm suggesting that instead of defining these constants, whatever is using them (probably controller logging code) should use `v1alpha1.Kind("ClusterProvisioner").String()` instead.